templates/vdbench_save.yml

```diff
@@ -150,16 +151,38 @@ steps:
       --query 'value[0].message' --output tsv |& tee ${cmdlog}
     grep -q ${cmdokmsg} ${cmdlog} || check_rc.sh 1 "Run azcopy .tar to storage account"
 
+    az resource lock delete -n DO_NOT_DELETE --resource ${STOR_ACCT_RESOURCE_ID}
+    check_rc.sh $rc "Delete Lock on storage account" || exit 0
```

Comment:
If for some reason the lock isn't here when this runs, will this command fail? Might not since "delete" calls usually return success when the requested resource doesn't exist. Just wanted to confirm either way because parallel pipeline runs could cause a race condition here.

Reply:
I thought we had talked about the race case, if lock isnt there it does return a 0 error code. 
That is why below there is that loop that will retry the delete if the private-endpoint delete fails.